test/features/user_records_steps_test.rb

```diff
@@ -20,16 +22,18 @@ def test_user_adds_a_step
   end
 
   def test_user_edits_a_step
-    app = Application.create(:company => "Basecamp", :status => "open")
-    step = app.steps.create(:kind => "feedback", :note => "They're hiring!")
+    user = create(:person)
+    app = user.applications.create(company: "Basecamp", status: "applied")
+    step = app.steps.create(kind: "feedback", note: "They're hiring!")
+    page.set_rack_session(user_id: user.id)
 
     visit application_path(app)
     assert page.has_content? "They're hiring!"
     assert page.has_content? "Feedback"
 
     click_link_or_button "edit_step_#{step.id}"
-    select('code_challenge', :from => 'step_kind')
-    fill_in 'step_note', :with => "They're NOT hiring!"
+    select('code_challenge', from: 'step_kind')
+    fill_in 'step_note', with: "They're NOT hiring!"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.